syncer/ddl.go

```diff
@@ -89,7 +89,7 @@ func (s *Syncer) parseDDLSQL(sql string, p *parser.Parser, schema string) (resul
 			if len(table.Schema) == 0 {
 				table.Schema = schema
 			}
-			ignore, err2 := s.skipDMLEvent(table, replication.QUERY_EVENT)
+			ignore, err2 := s.filterRowsEvent(table, replication.QUERY_EVENT)
```

Comment:
i think we should remove `case ast.DMLNode` from this func, beacuse we can make sure all passed in sql is DDL in row based binlog

cc @lance6716 @glorv

Reply:
About the change of `ddl.go`, I suggest discuss it in https://github.com/pingcap/dm/pull/2090. It had a huge change for `ddl.go`. You can read the draft changes and give me some suggestions for better.

Reply:
for now user may ignore precheck about ROW format binlog